docs/troubleshoot/security.md

```diff
@@ -41,11 +44,11 @@ cat <<EOF > ./trustpolicy.json
 }
 EOF
 notation policy import ./trustpolicy.json
+notation verify ghcr.io/ratify-project/ratify:v1.4.0
```

Comment:
Since `v1.4.0` is not released shall we go with the latest version `v1.3.1` instead?

Reply:
I was thinking that updates to the docs here are for the `next` version on the website, which corresponds to unreleased changes. The current v1.3.1 release is not signed so we don't want to give that as an example. When v1.4.0 is released, the published release images will be signed and match the docs.

Reply:
Should we hold on merging and publish this doc when v1.4.0 is released and signed?

Reply:
Either way works. In the past `next` version has consisted of changes matching current `dev` branch changes and will be officially released in next versioned docs.

Reply:
discussed at CC 12/9/24 that we will move forward and merge this